ocs_ci/deployment/deployment.py

```diff
@@ -646,6 +649,109 @@ def patch_default_sc_to_non_default(self):
                 f"--request-timeout=120s"
             )
 
+    def uninstall_osc(self):
+        ocp_obj = ocp.OCP()
+        provisioners = ['openshift-storage.rbd.csi.ceph.com',
+                        'openshift-storage.cephfs.csi.ceph.com',
+                        'openshift-storage.noobaa.io/obc']
+
+        # STEP 1:List the storage classes
+        sc_list = get_all_storageclass_names(all_ns=True)
+        for storage_class in sc_list:
+            sc = ocp.OCP(kind=constants.STORAGECLASS, resource_name=storage_class)
+            pro = sc.get('provisioner')
+            if pro != provisioners[0] and pro != provisioners[1] and pro != provisioners[2]:
+                sc_list.remove(storage_class)
+
+        # STEP 2: Query for PVCs and OBCs that are using the storage class provisioners listed in the previous step.
+        pvc_list = get_all_pvcs(namespace='all-namespaces')
+        pvcs_to_delete = []
+        for pvc in pvc_list:
+            for storage_class in sc_list:
+                if pvc.backed_sc == storage_class:
+                    pvcs_to_delete.append(pvc)
+                    break
+
+        # Removing monitoring stack from OpenShift Container Storage (edit config map with patch) # TODO
+        monitoring_obj = ocp.OCP(namespace=constants.MONITORING_NAMESPACE, kind='configmap')
+        monitoring_list = monitoring_obj.get('data').get('config.yaml')
+        for item in monitoring_list:
+            if item.get().get('volumeClaimTemplate').get('spec').get('storageClassName') in pvcs_to_delete:
+                item #TODO
+        monitoring_obj.patch(resource_name='cluster-monitoring-config', params='')
+
+        # Removing OpenShift Container Platform registry from OpenShift Container Storage
+        # Edit the configs.imageregistry.operator.openshift.io object and remove the content in the storage section.
+        ir_obj = ocp.OCP(namespace=constants.OPENSHIFT_IMAGE_REGISTRY_NAMESPACE)
+        if self.platform == constants.AWS_PLATFORM:
+            ir_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                         params=" '{\"spec\":{\"storage\":{}}}' ")
+
+        elif self.platform == constants.VSPHERE_PLATFORM:
+            ir_obj.patch(resource_name='configs.imageregistry.operator.openshift.io',
+                         params=" '{\"spec\":{\"storage\":{\"emptyDir\":{}}}}' ")
+
+        # Removing the cluster logging operator from OpenShift Container Storage
+        # Remove the ClusterLogging instance in the namespace.
+        ocp_obj.exec_oc_cmd("delete clusterlogging instance -n openshift-logging --wait=true --timeout=5m")
```

Comment:
can this be replaced with exec_oc_cmd()?

Reply:
what do you mean?

Reply:
oh sorry :) please ignore